compiler/qsc_codegen/src/qir.rs

```diff
@@ -20,7 +20,7 @@ use qsc_rir::{
 fn lower_store(package_store: &qsc_frontend::compile::PackageStore) -> qsc_fir::fir::PackageStore {
     let mut fir_store = qsc_fir::fir::PackageStore::new();
     for (id, unit) in package_store {
-        let package = qsc_lowerer::Lowerer::new().lower_package(&unit.package);
+        let package = qsc_lowerer::Lowerer::new().lower_package(&unit.package, &fir_store);
```

Comment:
Is there a reason for lowering and updating the package in two separate calls instead of using the `lower_and_update_package` function?

Reply:
I don't think you should refactor in this PR. After going through all the changes there are many places where using `lower_and_update_package` could be used but that can be done later. Closing this comment.